source/features/clone-branch.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import gitBranch from 'octicon/git-branch.svg';
+import insertTextTextarea from 'insert-text-textarea';
+import delegate, {DelegateEvent} from 'delegate-it';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import loadingIcon from '../libs/icon-loading';
+import {getRepoURL, getRepoGQL} from '../libs/utils';
+
+const getBranchBaseSha = async (branchName: string): Promise<string> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			ref(qualifiedName: "${branchName}") {
+				target {
+					oid
+				}
+			}
+		}
+	`);
+
+	return repository.ref.target.oid;
+};
+
+async function createBranch(newBranchName: string, baseSha: string): Promise<true | string> {
+	const response = await api.v3(`repos/${getRepoURL()}/git/refs`, {
+		method: 'POST',
+		body: {
+			sha: baseSha,
+			ref: 'refs/heads/' + newBranchName
+		},
+		ignoreHTTPStatus: true
+	});
+
+	return response.ok || response.message;
+}
+
+async function cloneBranch(event: DelegateEvent<MouseEvent, HTMLButtonElement>): Promise<void> {
+	const cloneButton = event.delegateTarget;
+	const branchElement = cloneButton.closest<HTMLElement>('[data-branch-name]')!;
+
+	const currentBranch = getBranchBaseSha(branchElement.dataset.branchName!);
+	let newBranchName = prompt('Enter the new branch name')?.trim();
+
+	const spinner = loadingIcon();
+	spinner.classList.add('mr-2');
```

Comment:
![sdf](https://user-images.githubusercontent.com/1402241/77261626-1ca6b800-6c90-11ea-814b-7a32fc7c0b0f.gif)

The spacing isn't right. Other than this, it seems to work well 👍

Reply:
I know this happened when we switched to using the position. I will try later tonight.

Reply:
oh.... Its supposed to be ml-2